quickstartKotlin/src/main/java/com/twilio/video/quickstart/kotlin/VideoActivity.kt

```diff
@@ -0,0 +1,670 @@
+            }
+        }
+    }
+
+    private fun retrieveAccessTokenfromServer() {
+        Ion.with(this)
+                .load("$ACCESS_TOKEN_SERVER?identity=${UUID.randomUUID()}")
+                .asString()
+                .setCallback({ e, token ->
+                    if (e == null) {
+                        this@VideoActivity.accessToken = token
+                    } else {
+                        Toast.makeText(this@VideoActivity,
+                                R.string.error_retrieving_access_token, Toast.LENGTH_LONG)
+                                .show()
+                    }
+                })
+    }
+
+    private fun configureAudio(enable: Boolean) {
+        if (enable) {
+            previousAudioMode = audioManager.mode
+            // Request audio focus before making any device switch
+            requestAudioFocus()
+            /*
+             * Use MODE_IN_COMMUNICATION as the default audio mode. It is required
+             * to be in this mode when playout and/or recording starts for the best
+             * possible VoIP performance. Some devices have difficulties with
+             * speaker mode if this is not set.
+             */
+            audioManager.mode = AudioManager.MODE_IN_COMMUNICATION
+            /*
+             * Always disable microphone mute during a WebRTC call.
+             */
+            previousMicrophoneMute = audioManager.isMicrophoneMute
+            audioManager.isMicrophoneMute = false
+        } else {
+            audioManager.mode = previousAudioMode
+            audioManager.abandonAudioFocus(null)
+            audioManager.isMicrophoneMute = previousMicrophoneMute
+        }
+    }
+
+    private fun requestAudioFocus() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            val playbackAttributes = AudioAttributes.Builder()
+                    .setUsage(AudioAttributes.USAGE_VOICE_COMMUNICATION)
+                    .setContentType(AudioAttributes.CONTENT_TYPE_SPEECH)
+                    .build()
+            val focusRequest = AudioFocusRequest.Builder(AudioManager.AUDIOFOCUS_GAIN_TRANSIENT)
+                    .setAudioAttributes(playbackAttributes)
+                    .setAcceptsDelayedFocusGain(true)
+                    .setOnAudioFocusChangeListener { }
+                    .build()
+            audioManager.requestAudioFocus(focusRequest)
+        } else {
+            audioManager.requestAudioFocus(null, AudioManager.STREAM_VOICE_CALL,
+                    AudioManager.AUDIOFOCUS_GAIN_TRANSIENT)
+        }
+    }
+
+    private fun createConnectDialog(participantEditText: EditText,
+                                    callParticipantsClickListener: DialogInterface.OnClickListener,
+                                    cancelClickListener: DialogInterface.OnClickListener,
+                                    context: Context): AlertDialog {
+        val alertDialogBuilder = AlertDialog.Builder(context)
```

Comment:
Dialog builder is a good candidate for [apply](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/apply.html) function